dracut/anaconda-lib.sh

```diff
@@ -214,7 +214,16 @@ set_neednet() {
     [ -z "$netroot" ] && >> /tmp/net.ifaces
 }
 
+wait_for_entropy() {
+    while [ "$(cat /proc/sys/kernel/random/entropy_avail)" -lt "10" ]; do
+        ENTROPY_AVAIL=$(cat /proc/sys/kernel/random/entropy_avail)
+        warn "Waiting for entropy ($ENTROPY_AVAIL) to be >= 10"
+        sleep 1
+    done
```

Comment:
This is quite unfortunate because AFAICT every single of those ``$(cat ...)``s, ``warn``s and ``sleep``s eat some entropy from the pool because kernel does process-memory-placement randomization. Does this really improve things? To me it seems like it must make the wait longer. Of course we at least report something, but I'm not sure that's really worth it.